src/amo/server/base.js

```diff
@@ -319,8 +319,25 @@ function baseServer(
       }
 
       let i18nData = {};
-      const { htmlLang } = pageProps;
+      const { htmlLang, assets } = pageProps;
       const locale = langToLocale(htmlLang);
+      const fontToPreload =
+        './src/fonts/woff2/Inter-roman-minimal-subset.var.woff2';
```

Comment:
Obviously hardcoding the name here isn't great. Thoughts on where/how to do that ? Note that it will be slightly dynamic when the same principle is applied to other locales as well.